### DISCUSSION TITLE
GitHub Discussions will become read-only on September 26th – please join us in Discord!

### DISCUSSION THREAD

**Discussion by carolinefrasca** [modular/modular#3507] [Announcements]
In an effort to centralize our community and ensure that all questions are answered as quickly as possible, GitHub Discussions for this repository will become read-only on September 26th and we ask that you share your questions, thoughts, and projects in [our Discord server](https://modul.ar/discord) instead.

We now have the [Modular Forum](https://forum.modular.com/) if you'd like to use.

Thank you for being part of our community and see you in Discord!

**Comment by Brian-M-J** [comment]
What happens to the discussions that have less than 10 comments? Are they deleted or just archived? I'd prefer the latter.

**Reply by carolinefrasca** [reply]
Hi @Brian-M-J, thanks for flagging this! We looked into this and decided to instead make GitHub Discussions read-only, so that you're able to read all of the past threads.
